.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: php
+
+php:
+  - 7.2.11
```

Comment:
I think it is worth settings up a build matrix to test this on multiple versions of Laravel to ensure that it still works correctly on 5.6 and 5.7.

Reply:
For example: https://github.com/SoapBox/Idempotency/blob/master/.travis.yml

In that file we test against 5.6 and 5.7. Since you have a dependency on 5.5 you'd probably want to test agains that too.

Reply:
Doing this will likely cause you to run into issue with resolving dependencies. I usually use the following to get around this.

```
"require-dev": {
    "phpunit/phpunit": "^6.0|^7.0",
    "orchestra/testbench": "^3.0"
}
```